src/btree/bt_vrfy.c

```diff
@@ -229,12 +229,19 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
 
     /* Check configuration strings. */
     WT_ERR(__verify_config(session, cfg, vs));
+    WT_ERR(__wt_config_gets(session, cfg, "checkpoint", &cfg_ckpt));
+    WT_ERR(__wt_config_gets(session, cfg, "dump_offsets", &cfg_dump_offsets));
+    is_specific_checkpoint = cfg_ckpt.len > 0;
 
-    /* Optionally dump specific block offsets. */
+    if (cfg_dump_offsets.len > 0 && is_specific_checkpoint)
+        WT_ERR_MSG(
+          session, ENOTSUP, "Providing both \'checkpoint\' and \'dump_offsets\' is not supported");
+
+        /* Optionally dump specific block offsets. */
```

Comment:
This change was made to comply with `clang-format` rules. This seems odd to me, so if you understand why `clang-format` suggested it or how to suppress it - please let me know.

Reply:
Please see this commit, the only change that this commit contains is what I am talking about, AFAIS `clang-format` misaligned the comment here: https://github.com/wiredtiger/wiredtiger/pull/11919/commits/afa9f3527e1078f50a62e1100b632b8ff8cb8483

Reply:
Looks like you found a bug indeed! If you add curly brackets to the `if` just above, then the comment is aligned correctly. But this against our coding standards - so we can just leave it as-is.